lib/user/getPlayersThumbnail.js

```diff
@@ -0,0 +1,47 @@
+// Includes
+var http = require('../util/http.js').func
+
+// Args
+exports.required = ['userIds', 'size']
+exports.optional = ['format', 'isCircular']
+
+// Define
+exports.func = function (args) {
+	if(!Array.isArray(args.userIds)){
```

Comment:
To make it easier (and to stop the many questions we'll no-doubt get) how would you feel about making `userIds: number | number[]`?
And then implementing a check, something like:
```js
if (!Array.isArray(args.userIds)) {
   if (typeof userIds === "number") {
      userIds = [userIds]
   } else {
     //error
    }
}
```

Reply:
Yeah I actually thought of that at first but then got confused while making the typings and totally forgot to create it